### PR TITLE
Update SearchRawTransaction calls for latest API.

### PR DESCRIPTION
**Pull request by davecgh**
**This pull request depends on btcsuite/btcd#487**

This pull request modifies the `SearchRawTransactions` and `SearchRawTransactionsVerbose` functions to work properly with the latest `searchrawtransactions` API in btcd.

In particular, this involves changing the return value from `[]*btcjson.TxRawResult` to `[]*btcjson.SearchRawTransactionResult` and adding the additional optional parameter which specifies whether or not to include information about the previous outputs.

### CODE REVIEW

File: rawtransactions.go [hunk 1]
```diff
@@ -595,10 +597,17 @@ func (r FutureSearchRawTransactionsVerboseResult) Receive() ([]*btcjson.TxRawRes
...
-func (c *Client) SearchRawTransactionsVerboseAsync(address btcutil.Address, skip, count int) FutureSearchRawTransactionsVerboseResult {
+func (c *Client) SearchRawTransactionsVerboseAsync(address btcutil.Address, skip,
+	count int, includePrevOut bool) FutureSearchRawTransactionsVerboseResult {
+
 	addr := address.EncodeAddress()
 	verbose := btcjson.Int(1)
-	cmd := btcjson.NewSearchRawTransactionsCmd(addr, verbose, &skip, &count)
+	prevOut := btcjson.Int(0)
+	if includePrevOut {
+		prevOut = btcjson.Int(1)
```
Comment (jrick) [inline rawtransactions.go]: *prevOut = 1 since it's already allocated

File: rawtransactions.go [hunk 2]
```diff
@@ -552,7 +552,9 @@ func (r FutureSearchRawTransactionsResult) Receive() ([]*wire.MsgTx, error) {
 func (c *Client) SearchRawTransactionsAsync(address btcutil.Address, skip, count int) FutureSearchRawTransactionsResult {
 	addr := address.EncodeAddress()
 	verbose := btcjson.Int(0)
-	cmd := btcjson.NewSearchRawTransactionsCmd(addr, verbose, &skip, &count)
+	prevOut := btcjson.Int(0)
```
Comment (jrick) [inline rawtransactions.go]: I'd use nil since 0 is the default value

File: rawtransactions.go (hunk 1 above)
Comment (davecgh) [inline rawtransactions.go]: Good point.  Will change.

File: rawtransactions.go (hunk 2 above)
Comment (davecgh) [inline rawtransactions.go]: Hmmm.  I typically like to be explicit in case the default changes at some point, but I can change it.

**Comment by jrick** [comment]
ok

**Pull request merged**